cli/src/cli/subcommand/interval/mod.rs

```diff
@@ -0,0 +1,45 @@
+/*
+ * This Source Code Form is subject to the terms of
+ * the Mozilla Public License, v. 2.0. If a copy of
+ * the MPL was not distributed with this file, You
+ * can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+use super::*;
+use structopt::clap::AppSettings::AllowLeadingHyphen;
+
+#[derive(Debug, StructOpt)]
+#[structopt(setting = AllowLeadingHyphen)]
+pub struct SubComInterval {
+    #[structopt(long, default_value = "2")]
+    count: u64,
+
+    #[structopt(allow_hyphen_values = true)]
+    low: i64,
+
+    #[structopt(allow_hyphen_values = true)]
+    high: i64,
```

Comment:
And doc comment each of the options explaining their purposes

Reply:
Can you let me know if what I added is suitable?